src/block/block_read.c

```diff
@@ -218,8 +217,10 @@ __wt_block_read_off(WT_SESSION_IMPL *session, WT_BLOCK *block, WT_ITEM *buf, uin
                   ENOSPC);
             }
         }
-        if (failures > 0 || !chunkcache_hit)
+        if (failures > 0 || !chunkcache_hit) {
```

Comment:
Could you swap these so that `!chunkcache_hit` is first.  I think it's more readable - I didn't see right away that the "normal" (non-chunkcache) case applies when the `if` is true.

Reply:
Thanks, good call - done in 7a41fee.